app/test.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Entry to all repository / application tests
+ */
+import Promise from 'bluebird';
+import _ from 'lodash';
+
+import app from '~/'
+import testFiles from './tests';
+
+// Test on update function
+// Import application object
+// Export promise of object with error and warning messages
+export function update(application) {
+  // Array of test functions
+  let loadedTests = [];
+
+  // Push all update functions to loadedTests array
+  for (let file in testFiles) {
```

Comment:
I'd probably have written this like:

```js
const loadedTests = testFiles
.filter(file => typeof file.update === Function)
.map(file => file.update(application));
```

...but that's just my opinion. I tend to prefer functional/immutable style unless it's really necessary to do something iteratively.